pyproject.toml

```diff
@@ -88,6 +88,8 @@ incremental = false
 strict_equality = true
 implicit_reexport = false
 no_implicit_optional = true
+# Ward tests redefine the name "_" a lot
+disable_error_code = ["no-redef"]
```

Comment:
It'll be slightly more configuration, but if we want to avoid this global disable, we can instead configure a separate mypy invocation for tests in `.pre-commit-config.yaml`.